extractor/src/main/scala/org/jetbrains/sbt/package.scala

```diff
@@ -2,6 +2,8 @@ package org.jetbrains
 
 import _root_.sbt.ProjectRef
 import _root_.sbt.Configuration
+import org.jetbrains.sbt.{structure => jb}
```

Comment:
(minor)
Redundant changes in this file.
Imports are not used, new line at the end of the file is unnecessary